clients/roscpp/src/libros/callback_queue.cpp

```diff
@@ -378,8 +384,11 @@ CallbackQueue::CallOneResult CallbackQueue::callOneCB(TLS* tls)
       }
       else
       {
-        tls->cb_it = tls->callbacks.erase(tls->cb_it);
```

Comment:
This was an artifact that I introduced when cleaning this patch up with comments today. Last night's test included this line.